File: src/main/java/olivermakesco/de/refmagic/RefinedMagicPotionUtil.java
package olivermakesco.de.refmagic;

import net.minecraft.entity.player.PlayerEntity;
import net.minecraft.item.Items;

public class RefinedMagicPotionUtil {
    public static final int cooldown = 20;
    public static void setCooldown(PlayerEntity player, int cooldown) {
        var manager = player.getItemCooldownManager();
        manager.set(Items.POTION, cooldown);
        manager.set(Items.SPLASH_POTION, cooldown);
        manager.set(Items.LINGERING_POTION, cooldown);
    }

    public static void setCooldown(PlayerEntity player) {
        setCooldown(player,cooldown);
    }
}

File: src/main/kotlin/olivermakesco/de/refmagic/RefinedMagicPotionUtil.kt
package olivermakesco.de.refmagic

import net.minecraft.entity.player.PlayerEntity
import net.minecraft.item.Items

object RefinedMagicPotionUtil {
    const val cooldown = 20
    fun setCooldown(player: PlayerEntity, cooldown: Int) {
        val manager = player.itemCooldownManager
        manager[Items.POTION] = cooldown
        manager[Items.SPLASH_POTION] = cooldown
        manager[Items.LINGERING_POTION] = cooldown
    }

    @JvmStatic
    fun setCooldown(player: PlayerEntity) {
        setCooldown(player, cooldown)
    }
}